cpp/log/cert_checker.cc

```diff
@@ -44,6 +45,29 @@ bool CertChecker::LoadTrustedCertificates(const std::string& cert_file) {
     return false;
   }
 
+  return LoadTrustedCertificatesFromBIO(bio_in);
+}
+
+bool CertChecker::LoadTrustedCertificates(const vector<string>& trusted_certs) {
+  string concat_certs;
+  for (vector<string>::const_iterator it = trusted_certs.begin();
```

Comment:
It's a shame we can't do lambdas - that loop would have been replaced with a single for_each line.